### ISSUE TITLE
Request dutch Bank Overboeking icon

### ISSUE THREAD

**Issue by Feggel** [vendocrat/PaymentFont#32]
Hi,
Could you make a variation of the "bank-transfer" icon.
Translating the "transfer" into "overboeking".
So it will say: Bank Overboeking.

You could also add:
Invoice icon. Translate into: "Factuur".

Thanks.

**Comment by jgautheron** [comment]
It would make sense to translate it to the most popular languages right away, or get rid of the text altogether so that we can add it manually.

**Comment by AlexanderPoellmann** [comment]
@jgautheron I basically agree that in some cases it would make sense to translate them right away. Nonetheless we're talking online shopping, and in that case I'm not so sure, that translating the bank transfer icon (as well as cash on delivery and cash on pickup) is absolutly necessary.

In my experience people who are shopping online actually do understand those three icons. On the other hand I chose to translate the invoice icon, as people didn't understand its meaning in German and Italian. But of course I'm open to other opinions and in case will add such translations as requested.

Anyway, what do mean with "get rid of the text altogether so that we can add it manually"? You mean something like a custom build via Fontello or via CSS?

**Comment by Feggel** [comment]
Yes usually they do understand them. But for an optimal shopping experience, translating the parts where you use text is best.

One quick fix though: I've said to add a bank-tranfser icon saying: bank overboeking, but the official correct Dutch translation would be: "Bankoverschrijving". So not: "Bank overboeking".

Wil you be adding these icons? Thanks
I really love them!

**Comment by jgautheron** [comment]
@AlexanderPoellmann They are not necessarily understood. Also there is a difference between end-users and the customers for who you're working on the website: usually they'll want everything translated.

A custom build that excludes text could be really helpful, since you cannot possibly cover all languages.

**Comment by AlexanderPoellmann** [comment]
@Feggel As mentioned before, I'm definitly considering adding translated icons depending on this discussion here!

@jgautheron Don't misunderstand me, with end-users I always mean customers. People who are working on the website are backend-users for me ;)

Nonetheless I agree with you guys, that translated icons will be best in terms of UX/shopping experience. On the other hand we'll have to define which languages to be added and which not, so that we don't blow PaymentFont with tons of duplicate icons.

**Comment by jgautheron** [comment]
Picking the languages will be hard. In order to be able to use the font on every eCommerce store my company handles, it would require about 30 languages.
It might be better to split the languages by regions, ex. "west-eu", "east-eu", "africa", "america", "asia", create a build for each, plus one which contains "all".

**Comment by Repalogic** [comment]
Perhaps, it is worth in this context to use the content attribute for internationalisation.

As I needed German translations in a project, I changed the pf style like this (incl. a border to make it more clear):

```
.pf {
            font-size: 30px;
            border: 1px solid #eee;
            padding: 5px;
            border-radius: 5px;
            color: #eee;
            margin: 2px 0;
            position: relative;
            height: 42px;
            width: 72px;
            display: inline-block;
            &:before {
                position: absolute;
            }
        }
```
Than, I was able to introduce own classes, e.g. pf-vorkasse:

```
.pf-vorkasse:before {
  content: "Vorkasse";
}
```` 

The position absolute is necessary as otherwise the own payment classes and the pf classes will not have the same top position (tested in Chrome an FF)

**Comment by WebBen** [comment]
Hi Alex,
"Barzahlen" is a payment provider, not a translation. Please see here: https://www.barzahlen.de/de/

**Comment by WebBen** [comment]
For the translations.... isn't it possible to have the onlineshop "Write" into the button? .... say you provide the Icon as for Invoice  but with a text attachement "invoice" that the shop can, depending on the choosen language translate?
